utils/index.js

```diff
@@ -211,6 +211,7 @@ module.exports.dataStoreDataDictionaryToTableSchema = (dataDictionary) => {
     type: dataDictionaryType2TableSchemaType[dataDictionary.type] || 'any'
   }
   if (dataDictionary.info) {
+    field.name = dataDictionary.info.title || dataDictionary.id
```

Comment:
@sagargg this looks wrong to me. `name` is the identifier in tableschema so it should be equal to `dataDictionary.id`, while `title` is the different thing.